### PR TITLE
ADD: Writing Documentation convert to .md

### PR DESCRIPTION
**Pull request by ghost**
# Description
Adding the .md version of the writing documentation workshop

## Type of change
- [x] Documentation (update or new)

# How Has This Been Tested?
Has been formatted with prettier
workshop was tested as a .pp with Documentation Team on Wednesday 06/04/22

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from ... on the Pull Request

### CODE REVIEW

File: docs/learning/training/writing-documentation.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+---
+
+## What is Documentation
+
+> “Documentation is one of the most important parts of a software project” – Eric Holscher, co-founder of Write the Docs.
+
+Documentation is official information provided by the company for our users and contributors. It is the preferred method of technical support and should provide instant clarity for many issues.
+
+Within Thoth Tech, all documentation will be written in markdown. You can find the guide [here](markdown-guide.md)
```
Comment (tancnle) [inline docs/learning/training/writing-documentation.md]:
```suggestion
Within Thoth Tech, all documentation will be written in markdown. You can find the guide [here](markdown-guide.md).
```

File: docs/learning/training/writing-documentation.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+We want our people to have the knowledge to make the most of our products
+
+**Respect:**
+We want our people to feel included and accepted for who they are
+
+**Educate:**
+We want to assist our people in their learning and provide them with the information they need
+
+**Guide:**
+we want to lead our people in a thoughtful manner towards their goals
```
Comment (tancnle) [inline docs/learning/training/writing-documentation.md]: **nitpick:** For consistency

```suggestion
We want to lead our people in a thoughtful manner towards their goals
```

File: docs/learning/training/writing-documentation.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+- Stay relevant to the title
+- Keep headlines and paragraphs short and scannable
+- Use second person
+- Simplicity and Clarity
+- Use Images and Screenshots to assist
+
+### Writing for Accessibility
+
+We must write for a diverse audience of readers who can interact with our content in different ways
```
Comment (tancnle) [inline docs/learning/training/writing-documentation.md]:
```suggestion
We must write for a diverse audience of readers who can interact with our content in different ways.
```

**Review by tancnle — APPROVED** [review]
Excellent effort @talzid ❤️ I have left some minor suggestions for your consideration. Let me know what you think. In the spirit of iteration, I approve it anyway.

**Comment by ghost** [comment]
Thank you @tancnle 
I will merge it, but I have given the Docs Team full permission to update as they see fit

**Pull request merged**